packages/core/src/sass/app/_common/layout/app-layouts.scss

```diff
@@ -56,33 +56,51 @@
 
 /** @define landscape-blocker; weak */
 .landscape-blocker {
-    display: none;
-
     /* enable temporary landscape blocker UI in landscape */
-    &--visible {
-        display: flex;
-        position: fixed;
-        top: 0;
-        left: 0;
-        overflow: hidden;
-        width: 100%;
-        height: 100%;
-        background: var(--general-main-1);
-        align-items: center;
-        justify-content: center;
-        flex-direction: column;
-        color: var(--text-prominent);
-        font-size: 1.6rem;
-        z-index: 9999;
-        font-weight: bold;
+    display: flex;
+    position: fixed;
+    top: 0;
+    left: 0;
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+    background: var(--general-main-1);
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    color: var(--text-prominent);
+    font-size: 1.6rem;
+    z-index: 9999;
+    font-weight: bold;
 
-        &__icon {
-            width: 59px;
-            height: 64px;
+    &__icon {
+        width: 59px;
+        height: 64px;
+    }
+    &__message {
+        padding: 1.6rem 11rem;
+        line-height: 1.4;
+    }
+
+    // checks if user is on desktop by checking if the primary input mechanism can hover over elements.
+    // pointer used to check if primary input mechanism is an accurate pointing device (mouse, touchpads, stylus).
+    @media (hover: hover) and (pointer: fine) {
+        display: none;
+    }
+
+    // checks if user is using any touch devices by checking if they can hover over elements or if their
+    // primary input mechanism has limited accuracy (remotes for tvs or game consoles).
+    @media only screen and (hover: none) and (pointer: coarse) {
+        @include mobile {
+            @media (max-aspect-ratio: 16/9) {
+                display: none;
+            }
```

Comment:
we check for aspect ratio instead of orientation on mobile only as it does not respect the soft-keyboard breaks css orientation rules. orientation also breaks on smaller devices